config/environments/development.rb

```diff
@@ -6,7 +6,9 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.cache_classes = true
+  config.action_controller.page_cache_directory = Rails.public_path + '/cache'
```

Comment:
Yes it was.  Here is the result.

bundle exec rubocop -D --format offenses --format progress || true
Inspecting 135 files
.......................................................................................................................................
--
0  Total